forms-thanks/appointments-thanks.html

```diff
@@ -1,32 +1,95 @@
-<!DOCTYPE html>

-<html lang="en">

-<head>

-  <title>Thank you: Research Consultation Appointment: MIT Libraries</title>

-  <link href="/css/header-semantic.css" rel="stylesheet" type="text/css">

-  <link href="/css/wp-libraries.css" rel="stylesheet" type="text/css">

-  <script type="text/javascript" src="/scripts/googleanalytics-async.js"></script>

-</head>

-<body>

-  <a class="semantic" href="#content">Skip to main content</a>

-  <div id="container">

-    <!--#include virtual="/includes/header-semantic.html"-->

-      <!-- breadcrumb - TODO: make this semantic -->

-      <div class="breadcrumb">

-        <span class="semantic">You are located in:</span>

-        <ul>

-          <li><a href="/"><span class="semantic"></span>MIT Libraries</a></li>

-          <li><a href="/search"><span class="semantic">in subsection </span>Research Support</a></li>

-          <li><a href="/forms-mit/appointments.html"><span class="semantic">in subsection </span>Research Consultation Appointment Form</a></li>

-          <li><span class="semantic">on page </span>Thank you!</li>

-        </ul>

-      </div>

-      <!-- page text -->

-      <h1>Research Consultation Appointment</h1>

-      <p>Thank you!</p>

-      <p>Your request for an appointment has been forwarded to the librarians handling your research topic. You will be contacted within one business day.</p>

-      <p>We look forward to meeting with you!</p>

-        <p>You will receive confirmation of our receipt of this form by email.</p>

-    <!--#include virtual="/includes/footer-semantic.html" -->

-  </div> <!-- /end container -->

-</body>

-</html>
\ No newline at end of file
+<!DOCTYPE html>
+<!--[if lte IE 9]><html class="no-js lte-ie9" lang="en"><![endif]-->
+<!--[if !(IE 8) | !(IE 9) ]><!-->
+<html lang="en-US" class="no-js">
+<!--<![endif]-->
+<head>
+<meta charset="UTF-8" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <title>Thank you: Research Consultation Appointment: MIT Libraries</title>
+
+  <!--#include virtual="/mitlib-style/inc/assets-site-head.html" -->
+
+  <!-- form specific stuff -->
```

Comment:
We can remove all these styles since they aren't applicable to the thanks pages.